File: uri/1 - Basics/1049/Main.java
import java.util.*;

class Main{
  public static void main(String [] args){

      Scanner scanner = new Scanner(System.in);
      String input  = scanner.next();
      String input2 = scanner.next();
      String input3 = scanner.next();
      
      if(input.equals("vertebrado")){
          if(input2.equals("ave")){
               if(input3.equals("carnivoro")){ 
                  System.out.printf("aguia\n");
               }
               if(input3.equals("onivoro")){ 
                  System.out.printf("pomba\n");
               }        
          }
          if(input2.equals("mamifero")){
               if(input3.equals("onivoro")){ 
                  System.out.printf("homem\n");
               }
               if(input3.equals("herbivoro")){ 
                  System.out.printf("vaca\n");
               }        
          }
      }
      
      if(input.equals("invertebrado")){
          if(input2.equals("inseto")){
             if(input3.equals("hematofago")){ 
                System.out.printf("pulga\n");
             }
             if(input3.equals("herbivoro")){ 
                System.out.printf("lagarta\n");
             }        
          }
          if(input2.equals("anelideo")){
             if(input3.equals("hematofago")){ 
                System.out.printf("sanguessuga\n");
             }
             if(input3.equals("onivoro")){ 
              System.out.printf("minhoca\n");
             }        
          }
      }

  }
}

File: uri/1 - Basics/1049/Main.go
package main

import (
	"fmt"
	"strings"
)

func strcmp(a, b string) bool {
	return strings.Compare(a, b) == 0
}

func main() {
	var input string
	var input2 string
	var input3 string

	fmt.Scanf("%s", &input)
	fmt.Scanf("%s", &input2)
	fmt.Scanf("%s", &input3)

	if strcmp(input, "vertebrado") {
		if strcmp(input2, "ave") {
			if strcmp(input3, "carnivoro") {
				fmt.Printf("aguia\n")
			}
			if strcmp(input3, "onivoro") {
				fmt.Printf("pomba\n")
			}
		}
		if strcmp(input2, "mamifero") {
			if strcmp(input3, "onivoro") {
				fmt.Printf("homem\n")
			}
			if strcmp(input3, "herbivoro") {
				fmt.Printf("vaca\n")
			}
		}
	}

	if strcmp(input, "invertebrado") {
		if strcmp(input2, "inseto") {
			if strcmp(input3, "hematofago") {
				fmt.Printf("pulga\n")
			}
			if strcmp(input3, "herbivoro") {
				fmt.Printf("lagarta\n")
			}
		}
		if strcmp(input2, "anelideo") {
			if strcmp(input3, "hematofago") {
				fmt.Printf("sanguessuga\n")
			}
			if strcmp(input3, "onivoro") {
				fmt.Printf("minhoca\n")
			}
		}
	}

}
